types/v56/types.go

```diff
@@ -241,6 +242,25 @@ type VAppNetworkConfiguration struct {
 	IsDeployed    bool                  `xml:"IsDeployed"`
 }
 
+// VAppNetworkType represents a vApp network configuration
+// Type: VAppNetworkType
+// Namespace: http://www.vmware.com/vcloud/v1.5
+// Description: Represents a vApp network configuration.
+// Since: 0.9
+type VAppNetwork struct {
```

Comment:
If possible, would like a comment explaining where `VAppNetwork` is used insetad of `VAppNetworkConfiguration`, as they both are very similar.

Reply:
added